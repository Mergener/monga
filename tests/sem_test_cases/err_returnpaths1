function foo(i : int): int {
    if (i < 4) {
        return 5;
    }
}