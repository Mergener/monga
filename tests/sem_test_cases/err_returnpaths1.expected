[Error] Module sem_test_cases/err_returnpaths1, line 5, column 2: Not all paths are returning a value.
Semantic errors were caught during analysis.
